modules/openedx/s3/course_discovery.tf

```
@@ -0,0 +1,52 @@
+data aws_iam_policy_document edxapp_course_discovery_s3_full_access {
+  count = var.enable_course_discovery ? 1 : 0
+  version = "2012-10-17"
+  statement {
+    actions = [
+      "s3:*",
+    ]
+    effect = "Allow"
+
+    resources = [
+      aws_s3_bucket.edxapp_course_discovery.arn,
+      "${aws_s3_bucket.edxapp_course_discovery.arn}/*",
+    ]
+  }
+}
+
+
+resource aws_s3_bucket edxapp_course_discovery {
+  count = var.enable_course_discovery ? 1 : 0
+  bucket = "${var.customer_name}-${var.environment}-edxapp-course-discovery"
```

It's not `var.customer_name` here (see https://github.com/open-craft/terraform-scripts/blob/main/modules/openedx/application/s3.tf#L8):
```suggestion
  bucket = "${var.client_shortname}-${var.environment}-edxapp-course-discovery"
```
Replace all occurrences in this file.

---

My bad